src/DBConnection.hpp

```diff
@@ -1,14 +1,32 @@
 #pragma once
 
+#include <string>
+
+struct Info
+{
+    Info() : serverVersion(""), protocolVersion(0) {};
+    ~Info() = default;
```

Comment:
@Morheit , why we need this default dtor?

Reply:
@malirod, Hm... Seems you're right. Maybe this usual default dtor will be redundant in this case. But I'm inclined to add `virtual` to this default dtor, because it's a good practice. If you don't mind, surely.

Reply:
@Morheit , I see no point in ctor (use default init for fields) and in dtor.

>> add virtual to this default dtor, because it's a good practice

virtual is needed if you're going to inherit from this class (with exceptions). So it's "bet practice" to not use "best practice" as an argument. You should clearly understand what is the reason of any "best practive" and you should be sure that such practice is applicable in your case.

Reply:
@malirod, got it. ctor and dtor removed